pubspec.yaml

```diff
@@ -3,15 +3,15 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: usage
-version: 3.4.2
+version: 4.0.0-nullsafety.1
 description: A Google Analytics wrapper for command-line, web, and Flutter apps.
 homepage: https://github.com/dart-lang/usage
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  path: ^1.4.0
+  path: ^1.8.0-nullsafety.3
```

Comment:
Curious: why the `.3` lower bound?

Reply:
This was just the latest available - I can drop to `1.8.0-nullsafety`.